Testcase                                                                                                                                                            Result
--------                                                                                                                                                            ------
[10] usb_passthrough dynamic_attach_on_boot                                                                                                                      ERROR (2)
        USB device 0781:5581 not found
